workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/GetAppPage/components/AccountSelection.tsx

```diff
@@ -29,16 +33,27 @@ const AccountSelection = ({onSelectAccount}: AccountSelectionProps) => {
 		const radioAccountList: RadioCardContent<Account>[] = [];
 
 		for (const accountBrief of userAccount.accountBriefs) {
-			const displayAccount = accountBrief.roleBriefs.reduce(
-				(display, roleBrief) => {
-					if (enabledAccountRoles.includes(roleBrief.name)) {
-						return true;
-					}
-
-					return display;
-				},
-				false
-			);
+			let displayAccount: boolean = false;
+			if (!accountBrief.roleBriefs.length) {
+				const accountInfo: Account = await getAccountInfo({
+					accountId: Number(accountBrief.id),
+				});
+				if (accountInfo.type === 'person') {
+					displayAccount = true;
+				}
+			}
+			else {
+				displayAccount = accountBrief.roleBriefs.reduce(
+					(display, roleBrief) => {
+						if (enabledAccountRoles.includes(roleBrief.name)) {
+							return true;
+						}
+
+						return display;
+					},
+					false
+				);
```

Comment:
I think the result is the same, can you test?

```suggestion
				displayAccount = accountBrief.roleBriefs.some(
					(roleBrief) => enabledAccountRoles.includes(roleBrief.name))
				);
```